src/lang/proc_macros/controller.rs

```diff
@@ -256,19 +251,25 @@ fn parse_proc_macro_response(response: RpcResponse) -> Result<ProcMacroResult> {
 }
 
 #[derive(Clone)]
-pub struct ProcMacroChannelsReceivers {
+pub struct ProcMacroChannels {
```

Comment:
Imo we should have separate structs for senders and receivers (can be substructs too)

Reply:
There is no need to do this, it would make it less clear only.

Reply:
I disagree, it is a weird struct with 2 pubs, but I can give it a pass

Reply:
I'd agree with Piotr M, but indeed this looks like a style thing. Let's stay this as-is to not bike-shed.